src/components/Main.js

```diff
@@ -1,12 +1,48 @@
-
+import React from'react';
+// import { Component } from 'react';
 import Hornedbeast from './Hornedbeast';
+import {Row, Container} from 'react-bootstrap';
+
 
-function Main() {
-  return(
-    <div>
-      <Hornedbeast />
-      <Hornedbeast />
-    </div>
-  );
+// class Main extends React.Component {
+//   render () {
+//     return(
+//       <div>
+//         <Container>
+//           <Row xs={1} md={5} className="g-4">
+//           {this.props.data.map( (value) => {
+//           <Col>
+//           <Hornedbeast 
+//           title={value.title}
+//           image_url={value.image_url}
+//           description={value.description}
+//           horns={value.horns}/>
+//           </Col>
+//           })}
+//           </Row>
+//         </Container>
+//       </div>
+//     );
+//   }
+// }
```

Comment:
Hey there, really nice work! 🥇 👍 

It looks like you changed up the `Main` component a bit, and are not using this code any longer. Try your best to remove old code before merging and submitting PR so things stay nice and clean :)